src/mca/gds/shmem/gds_shmem.c

```diff
@@ -1004,6 +1040,15 @@ shmem_segment_create_and_attach(
     }
     // Attach to the shared-memory segment.
     rc = shmem_attach(job, shmem_id, (uintptr_t)base_addr);
+    if (PMIX_UNLIKELY(PMIX_SUCCESS != rc)) {
+        PMIX_ERROR_LOG(rc);
```

Comment:
Just curious: do we really want to errorlog this here? The errorlog message won't mean anything to a user other than "something went wrong". If the intent is to warn them and then failover to the `hash` component, we probably should provide a "show help" instead so we can give them more info on what happened, note that we will failover and the implications of it, and how to turn "off" the message for future runs.

I'm not sure where your failover occurs, so perhaps this isn't the right place?

Reply:
Thank you, Ralph. You're right. I haven't implemented the failover yet because I want to try these changes so I don't fool myself into thinking one bit is working while another is not. I'll get the failover code in another pull request.

Reply:
Very good - thx!